lab_5_scrapper/scrapper.py

```diff
@@ -120,45 +247,106 @@ def __init__(self, full_url: str, article_id: int, config: Config) -> None:
         """
         Initializes an instance of the HTMLParser class
         """
-        pass
+        self._full_url = full_url
+        self._article_id = article_id
+        self._config = config
+        self.article = Article(self._full_url, self._article_id)
 
     def _fill_article_with_text(self, article_soup: BeautifulSoup) -> None:
         """
         Finds text of article
         """
-        pass
+        for text in article_soup.find_all('p')[:-2]:
+            self.article.text += text.text + '\n'
 
     def _fill_article_with_meta_information(self, article_soup: BeautifulSoup) -> None:
         """
         Finds meta information of article
         """
-        pass
+        title = article_soup.find('h1', attrs={'itemprop': 'headline'})
+        if not title:
+            self.article.title = 'WITHOUT TITLE'
```

Comment:
It is mandatory to extract title from the article source. It may not be straightforward, but it has to be done somehow. 
Exceptions are possible in extreme cases. Be prepared to prove that your case is extreme during lab defence.